src/Framework/App.Ref/src/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -73,6 +73,10 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       Private="false"
       ReferenceOutputAssembly="false" />
 
+    <ProjectReference Include="..\..\AspNetCoreAnalyzers\src\SourceGenerators\Microsoft.AspNetCore.App.SourceGenerators.csproj"
```

Comment:
@wtgodbe Can you sanity check my packaging changes here? I think it's correct given what we had to do in .NEt 8 for the `RequestDelegateGenerator` but let me know if I missed something.